docs/src/running-validator/validator-start.md

```diff
@@ -323,15 +323,23 @@ the validator to ports 11000-11020.
 
 The `--limit-ledger-size` parameter allows you to specify how many ledger
 [shreds](../terminology.md#shred) your node retains on disk. If you do not
-include this parameter, the validator will keep the entire ledger until it runs
-out of disk space.
-
-The default value attempts to keep the ledger disk usage under 500GB. More or
-less disk usage may be requested by adding an argument to `--limit-ledger-size`
-if desired. Check `solana-validator --help` for the default limit value used by
-`--limit-ledger-size`. More information about
```

Comment:
The "Check `solana-validator --help ...` bit is false; we intentionally don't set a default (will need to double check why that is the case) so removed it.